addons/sms/models/mail_message.py

```diff
@@ -24,6 +24,9 @@ def _compute_has_sms_error(self):
             message.has_sms_error = message in sms_error_from_notification
 
     def _search_has_sms_error(self, operator, operand):
-        if operator == '=' and operand:
-            return ['&', ('notification_ids.notification_status', '=', 'exception'), ('notification_ids.notification_type', '=', 'sms')]
-        raise NotImplementedError()
+        if operator != 'in':
+            raise NotImplementedError
+        return [('notification_ids', 'any', [
+            ('notification_status', '=', 'exception'),
+            ('notification_type', '=', 'sms'),
+        ])]
```

Comment:
It doesn't seems similar, it should be a OR inside the any right ?

Reply:
The code was wrong, the compute finds notifications that have type=sms and status=exception.